lib/favicons.php

```diff
@@ -36,7 +36,9 @@ function downloadHttp(&$url, $curlOptions = array()) {
 			CURLOPT_USERAGENT => FRESHRSS_USERAGENT,
 			CURLOPT_MAXREDIRS => 10,
 		));
-	curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);	//Keep option separated for open_basedir bug
+	if (!empty(ini_get('open_basedir'))) {
```

Comment:
I am not sure whether I actually want to fix this warning. It is only a warning; it still works when no redirection is needed. It is supposed to be fixed in PHP 5.6.0+ https://bugs.php.net/bug.php?id=65646

The things is that with this bug, the favicons might not be fetched correctly because HTTP redirections will not be followed, and this cannot easily be addressed from FreshRSS (otherwise we need to implement our own HTTP redirection following mechanism).

If we only want to get rid of the warning, a simple `@` could do the job.

Otherwise, please change to `ini_get('open_basedir') != ''` because `empty(function())` is only allowed since PHP 5.5 (and is slower anyway) http://php.net/manual/function.empty.php

For reference, this is what SimplePie is doing to support even older PHP versions:
https://github.com/simplepie/simplepie/blob/0edf39018709c0d66cfa6d9b972907bb92461efb/library/SimplePie/File.php#L112-L116

Reply:
What about only using the shutup operator @ instead?
We could write a log entry as well, which could potentially a benefit for some admins.
But at least I would make sure that this message does not show up in the global error log.

Reply:
Let's go for the `ini_get('open_basedir') != ''` version, which is more explicit.
Please add a comment that this is for PHP 5.6.0- (PHP bug 65646)

Reply:
It is when `open_basedir` is empty, that the curl option can be set, not the other way around

Reply:
brr, thats not my day, totally correct